### PR TITLE
Book model created and 2 books created through ruby's console

### PR DESCRIPTION
**Pull request by alanlamaswolox**
## Summary
Book model created and 2 examples created in the database through rails console to test if its working

## Trello
https://trello.com/c/RIUlnPeC/20-11-crear-modelo-de-book

### CODE REVIEW

File: app/models/book.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Book < ApplicationRecord
+  validates :gender, presence: true
```
Comment (nicozar95) [inline app/models/book.rb]: Don`t repeat code, just call them in one line

**Review by nicozar95 — CHANGES_REQUESTED** [review]
Great work! Small changes

File: app/models/book.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class Book < ApplicationRecord
+  validates :gender, :author, :image, :title, :editor, :year, presence: true
```
Comment (nicozar95) [inline app/models/book.rb]: Can you call the validation like: `validates %i[gender author image title editor year], presence: true`

**Review by nicozar95 — APPROVED** [review]
Really small change, if u would like to apply it

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -1,7 +1,6 @@
 # Default controller created by Rails
 class ApplicationController < ActionController::Base
   include DeviseTokenAuth::Concerns::SetUserByToken
-  protect_from_forgery with: :exception
```
Comment (nicozar95) [inline app/controllers/application_controller.rb:4]: Why isnt this change on the devise PR :thinking: ?

File: app/models/book.rb (hunk 2 above)
Comment (alanlamaswolox) [reply app/models/book.rb]: Nico, i changed this line but i wonder...what is this? I mean, what means this structure?

**Pull request merged**